infrastructure/helm-chart/charts/apps/charts/api/charts/api-admin/templates/deployment.yaml

```diff
@@ -51,8 +51,6 @@ spec:
             configMapKeyRef:
               name: kafka-config
               key: KAFKA_COMMIT_INTERVAL_MS
-        - name: SERVICE_NAME
```

Comment:
Why did you remove the service names?

Reply:
@pascal-airy Because I don't think that they are used. There used to be an implicit usage in the `ProducerHealthCheck`, but that was also unused so I removed both.